File: src/main/Java/boj_step5/boj11720.java
import java.io.*;

public class boj11720 {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(System.out));
        reader.readLine();
        String line = reader.readLine();
        int sum = 0;
        for (int i = 0; i < line.length(); i++) sum += line.charAt(i) - '0';
        writer.append(String.valueOf(sum));
        writer.close();
    }
}

File: src/main/kotlin/boj11720.kt
import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter

fun main() {
    val r = BufferedReader(InputStreamReader(System.`in`))
    val w = BufferedWriter(OutputStreamWriter(System.out))
    val n = r.readLine()!!.toInt()
    val s = r.readLine()
    var sum = 0
    for(i in 0 until s.length) sum += s.get(i).toString().toInt()
    w.write(sum.toString())
    w.close()
}